docs/monitoring-usage.rst

```diff
@@ -278,3 +278,86 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation for the `Creating Custom Metrics`_ for more information.
```

Comment:
Better to refer to the main landing page for [Using Custom Metrics](https://cloud.google.com/monitoring/custom-metrics/).